.idea/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM frolvlad/alpine-oraclejdk8:slim
```

Comment:
does this work on Java 10?

Reply:
wait, what is this doing in the idea folder?

Reply:
Good question...